test/algorithms/set_operations/union/union.cpp

```diff
@@ -430,6 +430,17 @@ void test_areal()
     TEST_UNION_REV(issue_566_a, 1, 0, -1, 214.3728);
     TEST_UNION_REV(issue_566_b, 1, 0, -1, 214.3728);
 
+#if ! defined(BOOST_GEOMETRY_USE_RESCALING) || defined(BOOST_GEOMETRY_TEST_FAILURES)
```

Comment:
I tested it with `side_robust` i.e. merging with https://github.com/boostorg/geometry/pull/873 and those tests are still failing, so probably not a side issue, and should be further investigated.

Reply:
Good to know, thanks